demo/Components/FileSelector.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div class="d-flex flex-row flex-wrap">
+    <v-img
+      v-for="(img, i) in images"
+      :key="'img-' + i"
+      class="ma-2 selectable"
+      max-height="100"
+      max-width="100"
+      :alt="img.alt"
+      :src="img.src" @click="selectImage(img)"
```

Comment:
Just a little reconfiguration to my code style.

```suggestion
      v-for="(img, i) in images"
      :key="'img-' + i"
      :alt="img.alt"
      :src="img.src"
      class="ma-2 selectable"
      max-height="100"
      max-width="100"
      @click="selectImage(img)"
```